src/ORM/Translatable/TranslatableSubscriber.php

```diff
@@ -68,11 +68,11 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
             return;
         }
 
-        if ($this->isTranslatable($classMetadata)) {
+        if ($this->isTranslatable($classMetadata, false)) {
```

Comment:
why are you passing false here, instead of directly putting false below? (see next comment)